helm/cluster/values.yaml

```diff
@@ -68,11 +84,49 @@ internal:
         memory: 384Mi
     registry: gsoci.azurecr.io
     workers: {}
+  ephemeralConfiguration: {}
```

Comment:
Can we also have similar to this for `HelmReleases` (e.g. Cilium) so that we can override their catalog and version in testing too.

Reply:
Good point, thanks for bringing it up. We should add that definitely! (just not sure if in this PR, as it's already really big change)

Reply:
Yeah no worries about it being a different PR. Just wanted to mention it somewhere as I went through this PR with Tinkerers and Turtles yesterday thinking about how we're going to go about testing default apps. 😁 Was very happy to find it already considered.

Reply:
@nprokopic Any objection to me creating an issue for this specific requirement (overriding HelmRelease version) so that I can track and reference it from our issues?

Reply:
Sorry for the late response, I was on a vacation. Feel free to create an issue (if you haven't already). Thanks!

Reply:
I had actually forgot but this was a good reminder :D Here you go - https://github.com/giantswarm/roadmap/issues/3440